scripts/test.sh

```diff
@@ -28,6 +28,19 @@
 # Capture what test we should run
 TEST_SUITE=$1
 
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+__proj_dir="$(dirname "$__dir")"
+
+# shellcheck source=scripts/common.sh
+. "${__dir}/common.sh"
+
```

Comment:
Add something like

`[[ "$TEST_TYPE" =~ ^(unit|integration)$ ]] || _error "invalid TEST_TYPE (value must be 'unit' or 'integration', recieved:${TEST_TYPE}"`